kubernetes/namespace.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: buildbarn
+  name: buildbarn
```

Comment:
The renaming of this file needs to be reflected in `run.sh` as it explicitly applies `bb-namespace.yaml`